javaslang/src/main/java/javaslang/collection/HashMultimap.java

```diff
@@ -25,10 +24,22 @@
  * @author Ruslan Sennov
  * @since 2.1.0
```

Comment:
This is all right, Ruslan did all the work and Multimap will be there in 2.1.0. I only copy-paste the impls and add additional generic parameters. Some fixes and a Builder-reimplementation on the way... It is a load of work, maybe I add my name...